demux/codec_tags.c

```diff
@@ -45,14 +45,55 @@ static const char *lookup_tag(int type, uint32_t tag)
 }
 
 // Corresponds to WMMEDIASUBTYPE_Base.
-static const unsigned char guid_ext_base[16] =
+static const unsigned char guid_ext_base[3][16] = {
     {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00,
-     0x80, 0x00, 0x00, 0xAA, 0x00, 0x38, 0x9B, 0x71};
+     0x80, 0x00, 0x00, 0xAA, 0x00, 0x38, 0x9B, 0x71},
+    {0x00, 0x00, 0x00, 0x00, 0x21, 0x07, 0xD3, 0x11,
+     0x86, 0x44, 0xC8, 0xC1, 0xCA, 0x00, 0x00, 0x00},
```

Comment:
can't find anything for 2107-d311-8644-c8c1ca000000 on the web

Reply:
See Section 5.5.1 of this dissertation; https://ria.ua.pt/bitstream/10773/11364/1/disserta%C3%A7%C3%A3o.pdf